client/components/FormattingBar/CommandMenu.tsx

```diff
@@ -6,14 +6,14 @@ import { Menu, MenuItem, MenuItemDivider } from 'components/Menu';
 
 type commandPropType = {
 	key: string;
-	title: string;
+	title: string | React.ReactNode;
```

Comment:
FYI: this can just be `React.ReactNode`, since strings are included in that type.